### PR TITLE
Display single movie Summary

### PR DESCRIPTION
**Pull request by sted1994**
## What does this PR do?
Allows users to click a movie poster to view the movie summary. Also, allows users to click the logo to return home.

## Where should the reviewer start?
Locally 

## How should this be manually tested?
Run npm test

### CODE REVIEW

**Review by michaelputnam67 — COMMENTED** [review]
Can we change the app divs to react Fragments?

File: src/App.js [hunk 1]
```diff
@@ -3,25 +3,43 @@ import Nav from "./Nav";
...
 
+
+
   render() {
     return (
       <div>
-        < Nav />
-        < PosterContainer movies={this.state.movies} />
+        < Nav returnHome={this.returnHome}/>
+        {(this.state.movieSummary) ? < MovieSummary movieToView={this.state.movieToView}/>: < PosterContainer movies={this.state.movies} showMovieSummary={this.showMovieSummary}/>}
```
Comment (emmacbd) [inline src/App.js:41]: nice ternary!

File: src/Nav.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ import React from "react";
 import "./css/nav.css"
 
 
-const Nav = () => {
+const Nav = ({returnHome}) => {
   return (
     <nav className="nav">
-      <h2>Rancid Tomatillos</h2>
+      <button onClick={() => returnHome()}><img src={require("./images/rancid-logo.png")}/></button>
```
Comment (emmacbd) [inline src/Nav.js:8]: Need to add cursor hover

File: src/Movie-Summary.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './css/movie-summary.css'
+
+const MovieSummary = (movieToView) => {
+  return (
+    <div className="movie-summary">
+      <img src={movieToView.movieToView.backdrop_path} alt="Backdrop for"/>
+      <h2>{movieToView.movieToView.title}</h2>
```
Comment (emmacbd) [inline src/Movie-Summary.js:8]: I'm guessing this info will display once we're making the fetch request to the individual movie api endpoint?

**Pull request merged**